enclave/src/rpc/mocks/rpc_gateway_mock.rs

```diff
@@ -97,6 +104,26 @@ impl RpcGateway for RpcGatewayMock {
         }
     }
 
+    fn validate_trusted_call_nonce(
```

Comment:
The mock function is here to decouple the unit test from other functions. With this mock however you are not decoupling - every unit test calling the RpcGateway Mock will also test the nonce storage, which is not the meaning of a unit test.

Reply:
The mocks were replaced by fixed data.